clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -812,7 +814,7 @@ else if (enableAutoCommit)
                     apiVersions);
 
             this.kafkaConsumerMetrics = new KafkaConsumerMetrics(metrics, metricGrpPrefix);
-
+            this.skipMetadataCacheUpdateUponUnassignment = config.getBoolean(ConsumerConfig.SKIP_METADATA_CACHE_UPDATE_UPON_UNASSIGN);
```

Comment:
nit: Add an empty line after it, so that this line would be considered an insertion instead of modification, which less likely introduces conflict in the future

Reply:
Good point